privacyidea/lib/tokens/tantoken.py

```diff
@@ -126,9 +126,15 @@ def update(self, param, reset_failcount=True):
             # init tokens with tans
             tans = param.get("tans").split()
             tan_dict = {k: v for k, v in enumerate(tans)}
+            # Avoid to generate TANs, since we get the from params
+            param["tantoken_count"] = 0
```

Comment:
As in the other case, the ``PaperTokenClass.update`` function will still generate 100 OTP values here because it reads the ``papertoken_count`` parameter.

Reply:
Will change this! thx!